test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.Old.cs

```diff
@@ -518,6 +518,14 @@ public void DnuPack_ResourcesNoArgs(string flavor, string os, string architectur
                         out stdError,
                         environment: null,
                         workingDir: workingDir);
+                    DnuTestUtils.ExecDnu(
```

Comment:
Since we're fixing this test, how about move this to the new test infrastructure all together?
/cc @JunTaoLuo